tools/gulp/tasks/example-module.ts

```diff
@@ -172,9 +173,16 @@ function parseExampleMetadata(fileName: string, sourceContent: string): ParsedMe
           if (decorator.expression.expression.text === 'Component') {
             for (const arg of decorator.expression.arguments) {
               for (const prop of arg.properties) {
-                const name = prop.name.text;
-                const value = prop.initializer.text;
-                meta[name] = value;
+                const propName = prop.name.text;
+
+                // Since additional files can be also stylesheets, we need to properly parse
+                // the styleUrls metadata property.
+                if (propName === 'styleUrls' && ts.isArrayLiteralExpression(prop.initializer)) {
+                  meta[propName] = prop.initializer.elements
+                    .map((literal: ts.StringLiteral) => literal.text);
+                } else {
+                  meta[propName] = prop.initializer.text;
```

Comment:
**Note**: All of this AST parsing is not typed properly, but this might be a different story (that's how it was before). 

<sup>Cleaning up the parsing here can be a follow-up and low-priority</sup>